src/test/ui/pattern/usefulness/integer-ranges/overlapping_range_endpoints.stderr

```diff
@@ -59,7 +59,6 @@ error: multiple patterns overlap on their endpoints
 LL |         0..=10 => {}
    |         ------ this range overlaps on `10_u8`...
 LL |         20..=30 => {}
-   |         ------- this range overlaps on `20_u8`...
```

Comment:
this information is lost

Reply:
Fixed, although it required a hack I'm not proud of....